exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/MarshalerUtil.java

```diff
@@ -107,6 +107,26 @@ private static int sizeRepeatedFixed64(ProtoFieldInfo field, int numValues) {
     return size;
   }
 
+  /**
+   * Returns the size of a repeated uint64 field.
+   *
+   * <p>Packed repeated fields contain the tag, an integer representing the incoming payload size,
+   * and an actual payload of repeated varints.
+   */
+  public static int sizeRepeatedUInt64(ProtoFieldInfo field, List<Long> values) {
```

Comment:
Think we need to update this one too for the PrimitiveLongList pattern

Reply:
Yeah, I wasn't sure about that since I was keeping it consistent with `sizeRepeatedFixed64` which takes in a `Long<List>`. Anyway, I've just pushed the change to make this one receive a `long[]`.